src/overlays/actors/ovl_En_Hg/z_en_hg.c

```diff
@@ -184,92 +201,93 @@ void func_80BCF4AC(EnHg* this, PlayState* play) {
         }
         if ((Math_Vec3f_DistXZ(&this->actor.world.pos, &this->actor.home.pos) > 200.0f) &&
             (Math_Vec3f_DistXZ(&player->actor.world.pos, &this->actor.home.pos) > 200.0f)) {
-            func_80BCF5F0(this);
+            EnHg_SetupChasePlayerWait(this);
         }
     }
 }
 
-void func_80BCF5F0(EnHg* this) {
-    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, 0);
-    this->actionFunc = func_80BCF634;
+void EnHg_SetupChasePlayerWait(EnHg* this) {
+    Actor_ChangeAnimationByInfo(&this->skelAnime, sAnimationInfo, HG_ANIM_IDLE);
+    this->actionFunc = EnHg_ChasePlayerWait;
 }
 
-void func_80BCF634(EnHg* this, PlayState* play) {
+void EnHg_ChasePlayerWait(EnHg* this, PlayState* play) {
     Player* player = GET_PLAYER(play);
 
     if (Math_Vec3f_DistXZ(&player->actor.world.pos, &this->actor.home.pos) < 200.0f) {
-        func_80BCF468(this);
+        EnHg_SetupMoveTowardsPlayer(this);
     }
 }
```

Comment:
I'm not sure what the intention in this name is, is it waiting before chasing Player or waiting while chasing?

Reply:
It is meant to be used to wait while chasing, particularly when Link goes up the stairs, and he waits for him to come back down.